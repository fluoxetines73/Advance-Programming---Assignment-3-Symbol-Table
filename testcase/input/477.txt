PRINT
INSERT b string
ASSIGN b 'HelloWorld'
RPRINT
BEGIN
LOOKUP b
END