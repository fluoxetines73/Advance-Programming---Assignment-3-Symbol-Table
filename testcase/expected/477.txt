
success
success
b//0
0